scripts/distribution/macOS-package/template/scripts/postinstall

```diff
@@ -82,35 +82,52 @@ function configureService() {
     fi
 }
 
-function configureServices () {
+# Uninstalls apps and support files related to a net.
+function uninstallNet() {
+    local netName=${1:?"Missing net name to uninstall"}
+    echo "-- Removing $netName node apps and support files"
+    rm -rf "/Applications/Concordium Node/Concordium Node Start $netName.app"
```

Comment:
You should check whether the folders exist before trying to delete them, as the script will fail due to `set -e` if they don't exist.

Reply:
Good point but there is no need to do that because
1- By the time this line executes this folder is supposed to be created 
2- `rm -rf` does not fail